website/docs/language/meta-arguments/for_each.html.md

```diff
@@ -22,10 +14,18 @@ pool of compute instances) without writing a separate block for each one.
 Terraform has two ways to do this:
 [`count`](/docs/language/meta-arguments/count.html) and `for_each`.
 
+> **Hands-on:** Try the [Manage Similar Resources With For Each](https://learn.hashicorp.com/tutorials/terraform/for-each?in=terraform/0-13&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
```

Comment:
Remove UTMs, put collection in configuration language instead of 0-13

```suggestion
> **Hands-on:** Try the [Manage Similar Resources With For Each](https://learn.hashicorp.com/tutorials/terraform/for-each?in=terraform/configuration-language) tutorial on HashiCorp Learn.
```